### ISSUE TITLE
Allow users to set email notification frequency

### ISSUE THREAD

**Issue by csnyulas** [protegeproject/webprotege#390]
We should support some kind of digest mode for email notifications as well as switching off email notifications. Getting separate emails for every change/comment will quickly overwhelm users, especially if they participate on an actively developed project.

WebProtege 1.0 had 4 delivery options: "never", "immediately" (which could be configured on the server to a frequency of every few minutes), "hourly", and "daily". This worked very well, and users have never complained.

**Comment by csnyulas** [comment]
Related to #71

**Comment by schtesch** [comment]
Yes, please add those email control functions and set the default to never please. Thanks!

**Comment by rodriguesk** [comment]
Even when access is revoked from the ontology people still get the comment emails. This seems like a security issue too.
